packages/url_launcher/android/src/main/java/io/flutter/plugins/urllauncher/UrlLauncherPlugin.java

```diff
@@ -43,28 +43,9 @@ public void onMethodCall(MethodCall call, Result result) {
     if (call.method.equals("canLaunch")) {
       canLaunch(url, result);
     } else if (call.method.equals("launch")) {
-      Intent launchIntent;
```

Comment:
Thanks for cleaning up here!
I'd rather keep it to a separate PR to reduce noise from this one (and also, I think looking on these lines and seeing 'universal ios links' in the blame line will be a bit confusing)